isis/src/base/apps/findimageoverlaps/findimageoverlaps.cpp

```diff
@@ -11,13 +11,13 @@ using namespace std;
 
 namespace Isis {
 
-  void findimageoverlaps(UserInterface &ui, Pvl *log) {
+  void findimageoverlaps(UserInterface &ui, bool threadedCalculate, Pvl *log) {
```

Comment:
Kind of a nitpick, but can this be changed to `useThreads`? More in line with boolean naming, I know it's just named after the member var but the member var name isn't that great.

Reply:
Can it be `usethread` since there is only one? ;)